extensions/amp-story/amp-story-ads.md

```diff
@@ -0,0 +1,176 @@
+# Advertising support in AMP Stories 
+
+## AMP Story Ad Principles
+
+The current ad ecosystem of banners and boxes doesn't integrate well with the rest of the content around it. Ads feel out of place, are slow and interruptive and don't feel native to the rest of the story experience.
+
+We'd like to establish the following principles for Story Ads:
```

Comment:
Unclear who "we" is.  Consider explicitly stating who it is or it might be better to write around it --> To improve the integration of ads in content, the following principles will apply to AMP story ads:

Reply:
done